webapp/packages/core-authentication/src/TeamInfoMetaParametersResource.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { injectable } from '@cloudbeaver/core-di';
+import { CachedMapAllKey, CachedMapResource, isResourceAlias, ResourceKey, resourceKeyList, ResourceKeyUtils } from '@cloudbeaver/core-resource';
+import { GraphQLService } from '@cloudbeaver/core-sdk';
+
+import type { TeamMetaParameter } from './TeamMetaParametersResource';
+import { TeamsResource } from './TeamsResource';
+
+@injectable()
+export class TeamInfoMetaParametersResource extends CachedMapResource<string, TeamMetaParameter> {
+  constructor(
+    private readonly graphQLService: GraphQLService,
+    private readonly teamsResource: TeamsResource,
+  ) {
+    super();
+
+    this.sync(this.teamsResource);
+    this.teamsResource.onItemDelete.addHandler(this.delete.bind(this));
+  }
+
+  protected async loader(param: ResourceKey<string>): Promise<Map<string, TeamMetaParameter>> {
+    const all = this.aliases.isAlias(param, CachedMapAllKey);
+    const teamsList: [string, TeamMetaParameter][] = [];
+
+    await ResourceKeyUtils.forEachAsync(param, async key => {
+      let teamId: string | undefined;
+
+      if (!isResourceAlias(key)) {
+        teamId = key;
+      }
+
+      const { teams } = await this.graphQLService.sdk.getTeamsListMetaParameters({
+        teamId,
+      });
+      const metaParameters = teams[0].metaParameters;
```

Comment:
this may produce "cant access metaParameters of undefined" please check that teams.length === 0 and throw an error "Team {teamId} not found"

Reply:
done